modules/RTC/RTCUtils.js

```diff
@@ -753,10 +759,12 @@ var RTCUtils = {
                 });
             } else {
                 try {
-                    logger.error('Browser does not appear to be WebRTC-capable');
+                    logger.error(
+                        'Browser does not appear to be WebRTC-capable');
                 } catch (e) {
                 }
```

Comment:
@hristoterezov, what is the purpose of the empty catch around logger.error?

Is the idea to ensure reject gets invoked in case logger.error throws? If that's the case, wouldn't a finally be more appropriate?